phpunit/tests/import.php

```diff
@@ -76,7 +98,7 @@ function test_small_import() {
 		$comment_count = wp_count_comments();
 		$this->assertEquals( 1, $comment_count->total_comments );
 
-		$posts = get_posts(
+		$posts = get_posts( // phpcs:ignore WordPress.VIP.RestrictedFunctions.get_posts_get_posts
```

Comment:
I've seen this ignore a couple of times in the test files - why not just exclude the sniff for the test directory from the ruleset ?

Reply:
This `ignore` can now be removed as VIP is no longer included. You may want to do a search against `phpcs:` to see if there are more ignores which can be removed.